packages/augur-ui/src/modules/portfolio/components/common/sidebar-nav.styles.less

```diff
@@ -62,6 +62,7 @@
   > div:nth-of-type(2) {
     padding: @size-24 @size-12;
     background-color: var(--color-page-background);
+    min-height: calc(100vh - @size-64);
```

Comment:
shouldn't this be @top-bar-height-mobile?

Reply:
Not really. That's the height of the filter's header. I thought about using that variable but if we ever change the height for the top bar, we'll be messing around with the filter's header.

![image](https://user-images.githubusercontent.com/10351013/95765290-04c46f00-0c88-11eb-8f84-47a00baf75c3.png)